### DISCUSSION TITLE
Need Help with Repeater Capabilities/Options

### DISCUSSION THREAD

**Discussion by poqdavid** [Mygod/VPNHotspot#605] [Q&A] [answered]
I have a question regarding the VPNHotspot app. I noticed that the app uses WPA2 for its Repeater feature. Could someone explain why VPNHotspot is using WPA2 for the repeater? Are there specific advantages or technical reasons behind this?

**Comment by Mygod** [comment, marked as answer]
It is powered by Wi-Fi direct which uses WPA2.

**Reply by poqdavid** [reply]
Thank you very much! 😊 I assume there is no way to force it. Also, since it’s WPA2, do you think it’s bad to use it? The repeater seems to work much better on my Samsung device.

**Reply by KT1iN** [reply]
Thanks for the question! I was wondering about this too ❤️
